vpr/src/place/place.cpp

```diff
@@ -49,16 +51,23 @@
 
 #include "placer_breakpoint.h"
 
+/*  define the reward function factor constant *
+ *  This factor controls the weight of bb cost *
+ *  compared to the timing cost in the agent's *
```

Comment:
say "RL agent's"
Explain what 0.8 means:  0.8 weight on BB cost and 1 on timing cost?  Or 0.8 weight on BB cost and 0.2 on timing cost? Or something else?

Reply:
Fixed.